Common/hipsolver_utils.hpp

```diff
@@ -46,6 +46,9 @@ inline const char* hipsolverStatusToString(hipsolverStatus_t status)
         case HIPSOLVER_STATUS_HANDLE_IS_NULLPTR: return "HIPSOLVER_STATUS_HANDLE_IS_NULLPTR";
         case HIPSOLVER_STATUS_INVALID_ENUM: return "HIPSOLVER_STATUS_INVALID_ENUM";
         case HIPSOLVER_STATUS_UNKNOWN: return "HIPSOLVER_STATUS_UNKNOWN";
+#if hipsolverVersionMajor * 10 + hipsolverVersionMinor >= 18
```

Comment:
(Comment on internal MR) I would make this explicit since it's not a guarantee that minor version is $\lt 10$

```suggestion
#if (hipsolverVersionMajor == 1 && hipsolverVersionMinor >= 8) || hipsolverVersionMajor >= 2
```